aggregator/follower_flush_mgr_test.go

```diff
@@ -410,8 +410,8 @@ func TestFollowerFlushTaskRun(t *testing.T) {
 		i := i
 		flusher := NewMockflushingMetricList(ctrl)
 		flusher.EXPECT().
-			DiscardBefore(gomock.Any()).
-			Do(func(beforeNanos int64) {
+			DiscardBefore(gomock.Any(), disAllowEagerForwarding).
+			Do(func(beforeNanos int64, _ eagerForwardingMode) {
```

Comment:
can verify the forwardingMode passed in here?

Reply:
Sure